tests/integration/exec.bats

```diff
@@ -476,3 +476,15 @@ EOF
 	[ "$status" -eq 0 ]
 	[[ "${lines[0]}" == "Cpus_allowed_list:	$all_cpus" ]]
 }
+
+@test "runc exec with RLIMIT_NOFILE" {
+	update_config '.process.capabilities.bounding = ["CAP_SYS_RESOURCE"]'
+	update_config '.process.rlimits = [{"type": "RLIMIT_NOFILE", "hard": 65536, "soft": 65536}]'
+
+	runc run -d --console-socket "$CONSOLE_SOCKET" test_busybox
+	[ "$status" -eq 0 ]
+
+	# issue: https://github.com/opencontainers/runc/issues/4195
+	runc exec test_busybox /bin/sh -c "ulimit -n"
+	[[ "${output}" == "65536" ]]
+}
```

Comment:
As @kolyshkin said in #4266, there isn't much point making a CI test for this -- because this is a race we would need to run this for many iterations to confirm whether the race still exists. Running it once just wastes CI time because even if the bug was re-introduced we would never catch it in a single CI run.

Reply:
OK, I'll remove it later, at this time, I used it to test some other issues.